Doc/library/unittest.rst

```diff
@@ -2274,9 +2274,12 @@ These must be implemented as class methods::
         def tearDownClass(cls):
             cls._connection.destroy()
 
-If you want the ``setUpClass`` and ``tearDownClass`` on base classes called
-then you must call up to them yourself. The implementations in
-:class:`TestCase` are empty.
+The implementations in :class:`TestCase` are empty.
+
+If you want the ``setUpClass`` and ``tearDownClass`` on a superclass
```

Comment:
```suggestion
If you want the ``setUpClass`` and ``tearDownClass`` on a base class
```
Is it a typo? Because your overriden method is the method on a superclass,  so maybe it is something wrong that 'if you want the method on a superclass called from your superclass method'

Reply:
I found a simpler way to make it clear normal inheritance rules apply, without referring to my original use case (chain of three classes).